### PR TITLE
update schema refs

### PR DESCRIPTION
**Pull request by tomkralidis**

### CODE REVIEW

File: core/openapi/ogcapi-records-1.yaml [hunk 1]
```diff
@@ -237,7 +237,7 @@ components:
                 default: 'http://www.opengis.net/def/uom/ISO-8601/0/Gregorian'
       landingPage:
         allOf:
-          - $ref: "#/components/schemas/common/link.yaml"
```
Comment (pvretano) [inline core/openapi/ogcapi-records-1.yaml:240]: This should be `#/components/schemas/common/link`.  It's supposed to be an internal reference.

Comment (tomkralidis) [reply core/openapi/ogcapi-records-1.yaml:240]: When do we use internal references vs. `$ref`?  I see both in this file.

File: core/openapi/ogcapi-records-1.yaml [hunk 2]
```diff
@@ -237,7 +237,7 @@ components:
...
+          - $ref: schemas/common/link.yaml
```
Comment (pvretano) [inline core/openapi/ogcapi-records-1.yaml:240]: Should be: "#/components/schemas/common/link"

**Review by pvretano — APPROVED** [review]
@tomkralidis, look good.

**Pull request merged**